src/test/java/com/bandwidth/sdk/model/CallTest.java

```diff
@@ -423,4 +438,41 @@ public void shouldFailGetCallById() throws Exception {
         Call.get(mockClient, "id1");
     }
 
+    @Test
+    public void shouldUpdateCall() throws Exception {
+        RestResponse response = new RestResponse();
+        response.setResponseText(
+                "{\n" +
+                "   \"to\": \"+11111111111\"," +
+                "   \"recordings\": \"https:\\/\\/api.catapult.inetwork.com\\/v1\\/users\\/calls\\/recordings\"," +
+                "   \"transcriptionEnabled\": false," +
+                "   \"direction\": \"in\"," +
+                "   \"events\": \"https:\\/\\/api.catapult.inetwork.com\\/v1\\/users\\/calls/events\"," +
+                "   \"chargeableDuration\": 360," +
+                "   \"state\": \"completed\"," +
+                "   \"from\": \"+22222222222\"," +
+                "   \"id\": \"{callId}\"," +
+                "   \"endTime\":\"2014-08-12T10:59:30Z\"," +
+                "   \"recordingEnabled\": false," +
+                "   \"startTime\": \"2014-08-12T10:54:29Z\"," +
+                "   \"activeTime\": \"2014-08-12T10:54:29Z\"," +
+                "   \"transcriptions\": \"https:\\/\\/api.catapult.inetwork.com\\/v1\\/users\\/calls\\/transcriptions\"" +
+                "}"
+        );
+        response.setContentType("application/json");
+        mockClient.setRestResponse(response);
+
+        Map<String, Object> updateParams = new HashMap<String, Object>();
+        updateParams.put("tag", "tag1");
+
+        JSONObject callJson = (JSONObject) new JSONParser().parse(NEW_CALL_JSON);
+
+        Call call = new Call(mockClient, callJson);
+
+        call.update(updateParams);
+
+        assertThat(mockClient.requests.get(0).name, equalTo("post"));
```

Comment:
extract local variable instead of duplicating the code to get the request:
``` 
Object request = mockClient.reqeuests.get(0);
assertThat(request.name, equalTo("post"));
// change other asserts accordingly
```